packages/api/cms-api/src/auth/resolver/auth.resolver.ts

```diff
@@ -12,7 +12,6 @@ import { AccessControlServiceInterface } from "../../user-permissions/user-permi
 import { GetCurrentUser } from "../decorators/get-current-user.decorator";
 
 interface AuthResolverConfig {
-    currentUser?: Type<CurrentUser>; // TODO Remove in future version as it is not used and here for backwards compatibility
```

Comment:
shall we make a codemod for this?

Reply:
i would have created a codemod for this: https://github.com/vivid-planet/comet-upgrade/pull/31.